etc/system_config.json

```diff
@@ -57,6 +70,46 @@
         }
     },
 
+    "DetectorsAndDescriptors": {
+        "CD_CSIFT_Image_example": {
+            "type": "ColorDetectAndDescribe_Image_csift",
+            "init": {
+                "work_directory": "/Users/jonathanowens/smqtk_benchmarking/detect_and_describe_csift",
+                "data_element_type": "DataMemoryElement",
+                "descriptor_element_factory": "MemoryDescriptorFactory",
+		"parallel": 1
+            }
+        }
+    },
+
+    "Quantization": {
+        "SKLearn_Cluster_MiniBatch_Kmeans": {
+            "type": "SKLearn_Cluster_MiniBatch_Kmeans",
+            "init": {
+                "work_dir": "/Users/jonathanowens/smqtk_benchmarking/quantization/kmeans/example_image",
+                "kmeans_k": 256,
+                "label": "labeslsl",
+                // "init_size": 768,
```

Comment:
For description purposes, explain why this parameter is commented out, or take it out of the example.